test/api_integration/apis/home/sample_data.ts

```diff
@@ -91,14 +95,15 @@ export default function ({ getService }: FtrProviderContext) {
 
             const resp = await es.search<{ timestamp: string }>({
               index: 'kibana_sample_data_flights',
-              body: {
-                sort: [{ timestamp: { order: 'desc' } }],
-              },
+              sort: [{ timestamp: { order: 'desc' } }],
```

Comment:
moved sort options out of body to match search expectation for es. see https://www.elastic.co/guide/en/elasticsearch/reference/8.9/search-search.html